packages/jsActions/native-mobile-resources/src/notifications/ClearAllDeliveredNotifications.ts

```diff
@@ -0,0 +1,34 @@
+// This file was generated by Mendix Studio Pro.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the import list
+// - the code between BEGIN USER CODE and END USER CODE
+// - the code between BEGIN EXTRA CODE and END EXTRA CODE
+// Other code you write will be lost the next time you deploy the project.
+import { NativeModules, Platform } from "react-native";
+import PushNotification from "react-native-push-notification";
+
+// BEGIN EXTRA CODE
+// END EXTRA CODE
+
+/**
+ * Clears all delivered notifications from notification tray
+ * @returns {Promise.<void>}
+ */
+export async function ClearAllDeliveredNotifications(): Promise<void> {
+    // BEGIN USER CODE
+    // Documentation https://github.com/zo0r/react-native-push-notification
+    const isIOS = Platform.OS === "ios";
+    if (
+        NativeModules &&
+        ((isIOS && !NativeModules.RNCPushNotificationIOS) || (!isIOS && !NativeModules.RNPushNotification))
+    ) {
+        return Promise.reject(new Error("Notifications module is not available in your app"));
+    }
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const RNPushNotification: typeof PushNotification = require("react-native-push-notification");
```

Comment:
For Mx9 it should be imported and not required.

Reply:
Reference -> https://github.com/mendix/widgets-resources/pull/415

Reply:
Does that mean we have to change all the other js action's require ?

Reply:
Exactly, just check the PR I sent you

Reply:
Fixed